podcasts/New Creation/New Preview/PlaylistPreviewViewModel.swift

```diff
@@ -141,7 +146,17 @@ class PlaylistPreviewViewModel: ObservableObject {
             }
         }
 
-        newPlaylistHasChanged = true
+        if operationQueue.operationCount > 0 {
+            operationQueue.cancelAllOperations()
+            episodes.removeAll()
+        }
+        let refreshOperation = PlaylistRefreshOperation(tableView: UITableView(), filter: newPlaylist) { [weak self] newData in
```

Comment:
This feels a bit hacky... using a new placeholder UITableView every time there is a change? Is this code something you want to update in the future? The operation?

Reply:
@SergioEstevao that method needs to be changed in general as the tableview into it is never used. It needs to be addressed separately